src/lifetracker/calendar/NamedEvent.java

```diff
@@ -47,66 +47,80 @@ public void setName(String name) {
 	}
 
 	/* (non-Javadoc)
-	 * @see lifetracker.calendar.EventI#getStartDate()
+	 * @see lifetracker.calendar.EventI#getStart()
 	 */
 	@Override
-	public Date getStartDate() {
-		return startDate;
+	public LocalDateTime getStart() {
+		return startDateTime;
 	}
 
 	/* (non-Javadoc)
-	 * @see lifetracker.calendar.EventI#setStartDate(lifetracker.calendar.Date)
+	 * @see lifetracker.calendar.EventI#setStar(java.time.LocalDateTime)
 	 */
 	@Override
-	public void setStartDate(Date startDate) {
-		this.startDate = startDate;
+	public void setStart(LocalDateTime start) {
+		this.startDateTime = start;
 	}
 
 	/* (non-Javadoc)
-	 * @see lifetracker.calendar.EventI#getEndDate()
+	 * @see lifetracker.calendar.EventI#getEnd()
 	 */
 	@Override
-	public Date getEndDate() {
-		return endDate;
+	public LocalDateTime getEnd() {
+		return endDateTime;
 	}
 
 	/* (non-Javadoc)
-	 * @see lifetracker.calendar.EventI#setEndDate(lifetracker.calendar.Date)
+	 * @see lifetracker.calendar.EventI#setEnd(java.time.LocalDateTime)
 	 */
 	@Override
-	public void setEndDate(Date endDate) {
-		this.endDate = endDate;
+	public void setEnd(LocalDateTime end) {
+		this.endDateTime = end;
 	}
 
 	/* (non-Javadoc)
 	 * @see lifetracker.calendar.EventI#getStartTime()
 	 */
 	@Override
-	public Time getStartTime() {
-		return startTime;
+	public LocalTime getStartTime() {
+		return startDateTime.toLocalTime();
 	}
 
 	/* (non-Javadoc)
-	 * @see lifetracker.calendar.EventI#setStartTime(lifetracker.calendar.Time)
+	 * @see lifetracker.calendar.EventI#getEndTime()
 	 */
 	@Override
-	public void setStartTime(Time startTime) {
-		this.startTime = startTime;
+	public LocalTime getEndTime() {
+		return endDateTime.toLocalTime();
 	}
 
 	/* (non-Javadoc)
-	 * @see lifetracker.calendar.EventI#getEndTime()
+	 * @see lifetracker.calendar.EventI#isToday()
 	 */
 	@Override
-	public Time getEndTime() {
-		return endTime;
+	public isToday() {
```

Comment:
You also need a return type, the file's giving errors in the IDE because of these two issues.